pkg/kube/client.go

```diff
@@ -186,7 +186,7 @@ func (c *Client) Get(namespace string, reader io.Reader) (string, error) {
 		}
 		for _, o := range ot {
 			if err := p.PrintObj(o, buf); err != nil {
-				log.Printf("failed to print object type '%s', object: '%s' :\n %v", t, o, err)
+				log.Printf("failed to print object type %d, object: %d :\n %v", t, o, err)
```

Comment:
- [ ] s/type %d, object: %d/type %s, object: %q/  
  I'm not sure what the type of variable "o" is here. If it's just a name, then the "%q" verb is appropriate. If it's a struct, then maybe "%v" would be better.